Makefile

```diff
@@ -737,7 +737,7 @@ test-wasm: $(WASM_MODULE_OUTPUTS)  ### Run engine tests
 
 .PHONY: bench
 bench: ## Run benchmarks in /benches
-	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "${DEFAULT_FEATURES}"
+	${MAYBE_ENVIRONMENT_EXEC} cargo bench --no-default-features --features "${DEFAULT_FEATURES}" | tee output.txt
```

Comment:
I might prefer to have this redirection in `benches.yml` given I don't think we'd want it to happen locally.

At the least, we should add it to `.gitignore`.

Reply:
Is there a benches.yml? I can't find any reference to this in the cargo bench docs?

Reply:
Sorry, I meant in the GA workflow file `.github/workflows/benches.yml`. We could redirect the output there since the GA action is technically the thing that needs it and avoid creating the file locally when `make bench` is run. Not a big deal though, I'm fine with creating the file and `.gitignore`ing it as you have here.